client/components/index.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, View } from 'angular2/core';
+import { Navbar } from './navbar/index';
+import { Home } from './home/index';
+import { About } from './about/index';
+import { RouteConfig, Route, ROUTER_DIRECTIVES } from 'angular2/router';
+
+@Component({
+    selector: "app"
+})
+
+@View({
+    directives: [Navbar, ROUTER_DIRECTIVES],
+    styles: [require('!raw!sass!./index.scss')],
+    template: require('./index.html')
+})
+
+@RouteConfig([
```

Comment:
please use provideRouter(). on my mind, it's a bad practice use RouteConfig Decorator, but it's up on you